### ISSUE TITLE
loader from cmd & bypass defender

### ISSUE THREAD

**Issue by b4sh1t1** [optiv/ScareCrow#22]
Hello;

Wonder if there any chance to bypass AV or Defender with this loader ? 

Looks pretty cool but unfortunately i did try to create it with -sandbox; -console; nothing work..

./ScareCrow -I payload.bin -etw -domain www.microsoft.com

also consider if there undetected powershell or command to download the loader from cmd only ?

Thanks.

**Comment by Tylous** [comment]
I am sorry can you rephrase the question? This framework is designed to bypass EDRs it does have several commands pre-built into the delivery options to download the loader. I would avoid powershell as it's easily detected.
